### DISCUSSION TITLE
Allow facilitator to hand off facilitator role to another user

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#5463] [Feedback]
A [user](https://app.hubspot.com/contacts/3888472/contact/9913651) 🔒  writes: 

> I thought I could transfer the facilitator but couldn’t figure out how

Facilitator's sometimes want to give the role away, as opposed to asking someone else to take it. 

Related to issue [5378](https://github.com/ParabolInc/parabol/issues/5378)

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/62113601) 🔒 writes:

> How come I can’t change the facilitator on this newly created board?
